src/DogStatsd.php

```diff
@@ -555,4 +568,21 @@ private function eventUdp($vals)
 
         return null;
     }
+
+    /**
+     * Normalize the value witout locale consideration before queuing the metric for sending
+     *
+     * @param string $value The value to normalize
+     *
+     * @return string Formatted value
+     */
+    protected function normalizeStat($value) {
+      // Controlls the way things are converted to a string.
+      // Otherwise localization settings impact float to string conversion (e.x 1.3 -> 1,3 and 10000 => 10,000)
+      // If one would rather control formatting themselves adn dont want to override the class, they can pass in a string
+      if (!is_float($value)) {
+        return $value;
+      }
```

Comment:
Could we always return a string (even for non float number) ?

Reply:
Sure, only floats have the formatting issue in php but for consistency we can handle all types like this. The only thing that really changes is that all types will be padded to whatever precision is configured. I.e. 2 => 2.00, 0.026 => 0.03